db/seeds/searchSeed.js

```diff
@@ -0,0 +1,51 @@
+var Search=require('../models/model');
+const mongoose =require('mongoose');
+mongoose.connect('mongodb://localhost/fetcher');
+const db = mongoose.connection;
+
+mongoose.connection.collections['searches'].drop( function(err) {
+  console.log('collection dropped');
+});
+
+//helper function
+
+
+var getNextId = function () {
+
+  return nextId++;
+};
+var titleArr=['shinningServices','cody barkery','hailey farmes', 'space station','pardox interactive','Valve','google','facebook','linkdin','amazon','gloria fish hunting','nike'];
+var catagoriesArr=['online','parking','cooking','food services','game stops','bomb maker','fantasy land'];
+var randomNumber=(arr)=>{

```

Comment:
`randomElement` might be a more appropriate name for this function because a random element is being returned rather than a random number.

Reply:
marked